tests/pipeline_operations_test.py

```diff
@@ -1,12 +1,47 @@
 import unittest
+import pyspark
 
 import pipeline_dp
 
+
 class PipelineOperationsTest(unittest.TestCase):
   pass
 
+
+class SparkRDDOperationsTest(unittest.TestCase):
+  @classmethod
+  def setUpClass(cls):
+    conf = pyspark.SparkConf()
+    SparkRDDOperationsTest.sc = pyspark.SparkContext(conf=conf)
+
+  def test_sample_fixed_per_key(self):
+    spark_operations = pipeline_dp.SparkRDDOperations()
+    data = [(1, 11), (2, 22), (3, 33), (1, 14), (2, 25), (1, 16)]
+    dist_data = SparkRDDOperationsTest.sc.parallelize(data)
```

Comment:
would self.sc work?

Reply:
No, it wouldn't because `setUpClass(cls)` is annotated in the parent class as @classmethod so we can't access and initialize `self.sc` in `setUpClass(cls)`.